auraed/src/bin/main.rs

```diff
@@ -29,6 +29,34 @@
 \* -------------------------------------------------------------------------- */
 
 #![warn(clippy::unwrap_used)]
+#![deny(bad_style,
+        const_err,
+        dead_code,
+        improper_ctypes,
+        non_shorthand_field_patterns,
+        no_mangle_generic_items,
+        overflowing_literals,
+        path_statements,
+        patterns_in_fns_without_body,
```

Comment:
This is [deny-by-default](https://doc.rust-lang.org/rustc/lints/listing/deny-by-default.html), so doesn't need to be included